config/application.rb

```diff
@@ -32,6 +32,24 @@ class Application < Rails::Application
 
     config.autoload_paths << Rails.root.join('lib').to_s
 
+    config.exceptions_app = routes
+
+    ActionDispatch::ExceptionWrapper.rescue_responses.merge!(
```

Comment:
Curious if we could add the contents of `config/initializers/http_rescue_responses.rb` in here, I had used it in the past to globally wrap `ActionController::ParameterMissing` and respond with 422 (to handle blank params).

Here's the contents, should be able to just add a case here right?
`ActionDispatch::ExceptionWrapper.rescue_responses["ActionController::ParameterMissing"] = :unprocessable_entity`

Reply:
Yup, for sure. Good find.